src/fonts/petaluma.ts

```diff
@@ -0,0 +1,9 @@
+import { PetalumaFont } from "./petaluma_glyphs";
```

Comment:
Any ideas on how we can reduce the number of steps to add fonts?

Reply:
I made this intermediate object in order to get one file per font (`vexflow-font-bravura-min.js`) and not two: one for the metrics and one for the glyphs. 
Another solution would be to place metrics and glyphs in the same file but it would be a big one.

Reply:
okay, gotcha